tests/columns/test_datetime.py

```diff
@@ -149,6 +149,31 @@ def test_use_client_timezone(self):
                 inserted = self.client.execute(query, settings=settings)
                 self.assertEqual(inserted, [(self.dt, ), (self.dt, )])
 
+    def test_insert_integers(self):
+        settings = {'use_client_time_zone': True}
+
+        with self.patch_env_tz('Europe/Moscow'):
+            with self.create_table('a DateTime'):
+                self.client.execute(
+                    'INSERT INTO test (a) VALUES', [(1530211034, )],
+                    settings=settings
+                )
+
+                query = 'SELECT toUInt32(a), a FROM test'
+                inserted = self.emit_cli(query, use_client_time_zone=1)
+                self.assertEqual(inserted, '1530211034\t2018-06-28 21:37:14\n')
+
+    def test_insert_integer_bounds(self):
+        for ts in [0, 1, 1500000000, 2**32-1]:
+            with self.create_table('a DateTime'):
+                self.client.execute(
+                    'INSERT INTO test (a) VALUES', [(ts, )]
```

Comment:
Great, but let's insert whole values in one insert: `'INSERT INTO test (a) VALUES', [(0, ), (1, ), (1500000000, ), (2**32-1, )]`. It will be a little more readable.